db/migrate/20191119223120_add_is_oldest_accepted_to_form_versions.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddIsOldestAcceptedToFormVersions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :form_versions, :is_oldest_accepted, :boolean, null: false, default: true
+
+    # Revert the default before setting a single one to true (otherwise validation fails).
+    FormVersion.all.each { |fv| fv.update!(is_oldest_accepted: false) }
+    FormVersion.all.each { |fv| fv.update!(is_oldest_accepted: fv.is_current) }
```

Comment:
Nice approach. Another way to do this is to do null: true initially and then change_column_null after you do the update.